src/Playwright.Tests.TestServer/SimpleServer.cs

```diff
@@ -215,6 +169,66 @@ public SimpleServer(int port, string contentRoot, bool isHttps)
                 .Build();
         }
 
+        public async Task ServeFile(HttpContext context)
```

Comment:
this is very similar to upstream: https://cs.github.com/microsoft/playwright/blob/19010a5ffa23cb0df84af5ce1eb1a8c73cac7fb3/utils/testserver/index.js#L276